src/main/java/io/seqera/migtool/MigRecord.java

```diff
@@ -15,10 +15,23 @@
  */
 class MigRecord implements Comparable<MigRecord> {
 
+    enum Language {
+        sql,
+        groovy;
```

Comment:
Let's use uppercase that's more idiomatic, here we don't need to care about ser-deser of those names

Reply:
:+1:. Done.